sdcm/cluster.py

```diff
@@ -2325,6 +2326,9 @@ def create_swap_file(self, size=1024):
         if "sct_configured_swapfile" not in result.stdout:
             self.log.warning("Swap file is not used on loader node %s.\nError details: %s", self, result.stderr)
 
+    def set_hostname(self):
+        self.log.warning('Method is not implemented for %s' % self.__class__.__name__)
```

Comment:
next time : `raise NotImplementedError('Method is not implemented for %s' % self.__class__.__name__)` since we will not know when to implement it when adding new backend